api/v1alpha1/openapi.yaml

```diff
@@ -3461,12 +3461,23 @@ components:
         gRPCEndpoint:
           type: string
           description: The gRPC endpoint for the console connection.
-        sessionID:
-          type: string
-          description: The session ID for the console connection.
+        sessions:
+          type: array
+          description: A list of console sessions for the console connection.
+          items:
+            $ref: '#/components/schemas/ConsoleSession'
```

Comment:
I was looking into doing it the other way around. List of sessions where each session can have a different endpoint too. Let me try to post an alternative, I had it 60% ...